static/app/views/explore/hooks/useGetSavedQueries.tsx

```diff
@@ -0,0 +1,54 @@
+import type {Actor} from 'sentry/types/core';
+import {useQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+import useOrganization from 'sentry/utils/useOrganization';
+
+export type SavedQuery = {
+  createdBy: Actor;
+  dateAdded: string;
+  dateUpdated: string;
+  end: string;
+  environment: string[];
+  fields: string[];
+  id: number;
+  interval: string;
+  lastVisited: string;
+  mode: string;
+  name: string;
+  orderby: string;
+  projects: number[];
+  query: string;
+  queryDataset: string;
+  range: string;
+  start: string;
+  // Can probably have stricter type here
+  visualize: Array<{
+    chartType: number;
+    yAxes: string[];
+  }>;
+};
```

Comment:
Does this match some sort of table schema? Perhaps you can add a comment explaining where these fields come from?